src/Core/src/Platform/iOS/WrapperView.cs

```diff
@@ -324,6 +324,8 @@ void SetShadow()
 				shadowLayer.ClearShadow();
 			else
 				shadowLayer.SetShadow(Shadow);
+
+			Layer.Mask = shadowLayer;
```

Comment:
Why does the shadow set the mask on the Layer, but the clipping/SetMask doesn't?

Why do we even have a mask layer? The code looks to be applying a mask to a layer that is then added as a sub layer (maybe since the code looks like it only works if the sub layer has a Delegate). So, we are creating, clipping and then adding the clipped as a sub layer... Why are we not just clipping the main Layer directly in the clip?

Reply:
@kubaflo @mattleibow I've just implemented a cleaner solution in #26789 which also solves an issue where setting `Clip = null` would hide all the content (due to an empty `Path`).